internal/fs/inode/file_test.go

```diff
@@ -1028,20 +1028,73 @@ func (t *FileTest) TestUnlinkLocalFile() {
 	assert.Equal(t.T(), "gcs.NotFoundError: object test not found", err.Error())
 }
 
-func (t *FileTest) TestReadLocalFileWhenStreamingWritesAreEnabled() {
-	// Create a local file inode.
-	t.createInodeWithLocalParam("test", true)
-	t.in.writeConfig = getWriteConfig()
-	err := t.in.Write(t.ctx, []byte("hi"), 0)
-	assert.Nil(t.T(), err)
-	assert.Equal(t.T(), int64(2), t.in.bwh.WriteFileInfo().TotalSize)
+func (t *FileTest) TestReadFileWhenStreamingWritesAreEnabled() {
```

Comment:
```suggestion
func (t *FileTest) TestReadFileThrowsErrorWhenStreamingWritesAreInProgress() {
```

Reply:
For local file when streaming writes are enabled, we dont support reads irrespective of whether write happened or not. Thats why not calling as in progress